### DISCUSSION TITLE
Welcome to CoreWCF Discussions!

### DISCUSSION THREAD

**Discussion by mconnew** [CoreWCF/CoreWCF#266] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Share annoyances that you wish WCF/CoreWCF did differently.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about your interest in CoreWCF. Are you looking to use it, or contribute new features?

**Comment by BradBarnich** [comment]
What is the best way to support CoreWCF? 

financially/sponsorship? 
picking up issues?

This project will be essential to my business, as there are many healthcare industry standard protocols that are SOAP based.

**Reply by birojnayak** [reply]
@BradBarnich thanks for your interest on this project...
regarding financially/sponsorship? I will leave to @mconnew to comment ... 
Picking up issues ? Definitely yes, we have lots issues https://github.com/CoreWCF/CoreWCF/labels/help%20wanted ... so we would love some more developers to contribute as we plan on delivering more features in 2021...

**Reply by mconnew** [reply]
There isn't currently a need for direct financial sponsorship at the moment as the .NET Foundation is sponsoring this project. Feel free to donate to the .NET Foundation under the Members menu on their [website](https://dotnetfoundation.org/).  

We don't have issues for everything that needs to be done because the feature set for WCF is massive and it would take a long time to write them all up, especially as so many of them are obscure. I've been working on the WCF codebase for nearly 10 years now and I still occasionally discover features I didn't know about. The best way you can contribute is to test it out, see what features are missing that you need and open an issue if there isn't one already. If you are interested in working on implementing it, I can give guidance on where to start. Also please vote on features in the pinned issue at the top of the issues page. If your needed feature isn't there, add a comment with a similar format to the existing ones. I can fix it up after if it's not quite right.  

If all the features you need are there and you still want to contribute, check the help wanted like Biroj mentioned. Even if you have a feature you need but you feel it's too big a chunk for you to take on (WCF codebase can be intimidating), then picking up a help wanted issue frees up time for others to take on needed features.

We also have just started monthly calls to discuss CoreWCF. You unfortunately just missed out first call that happened a few hours ago. They happen monthly on the second Thursday of each month. The details are in the wiki.

**Comment by JVSFugitive** [comment]
Is there an example for using a WebServiceHost class to create a REST interface utilizing CoreWCF? Is this even possible? 

I am trying to port an application to .NET 5.0 but the namespace WebServiceHost cannot be found under CoreWCF.
